securedrop-admin

```diff
@@ -0,0 +1,159 @@
+#!/usr/bin/env python2
+"""
+SecureDrop Admin Toolkit.
+
+For use by administrators to install, maintain, and manage their SD
+instances.
+"""
+
+import argparse
+import subprocess
+import os
+import sys
+import getpass
+import logging
+
+sdlog = logging.getLogger('SD')
+
+SD_DIR = os.path.dirname(os.path.realpath(__file__))
+ANSIBLE_PATH = os.path.join(SD_DIR, "./install_files/ansible-base")
+SITE_CONFIG = os.path.join(ANSIBLE_PATH, "group_vars/all/site-specific")
+VENV_DIR = os.path.join(SD_DIR, ".venv")
+VENV_ACTIVATION = os.path.join(VENV_DIR, 'bin/activate_this.py')
+
+
+def logger_setup(verbose=False):
+    """ Configure logging handler """
+    # Set default level on parent
+    sdlog.setLevel(logging.DEBUG)
+    level = logging.DEBUG if verbose else logging.INFO
+
+    stdout = logging.StreamHandler(sys.stdout)
```

Comment:
Seems like these comments aren't taking into context the Slack conversations we've had as a team (absent Garrett). I made it clear several times that I'm not expecting all my comments to be addressed. I was just being thorough and verbose both as due diligence and so that Mike, who doesn't write a lot of Python code, gets some good feedback and things to consider for future.

For example, let's look at the comment that really started this thread:

> I would recommend using logging.Logger in the future. You can set it to print to sys.stdout.

As I tried really hard to make clear here and on Slack, I wasn't expecting this change to be made. It was basically a "next time try this" #protip. So, if you're reading my comments as demanding all these little nits being fixed, I can see why you would think I'm being over the top or counterproductive to "agile." In Slack I specifically noted days ago that I have been ready to approve this PR, regardless of what gets addressed, once Mike or other team members deemed it is ready and tell me to do so.

Now, I feel like I gotta address this too to clarify some more things:

>I have to push back here buddy. I've already made a number of changes to the output at your request. You already brought up a nit that I should be utilizing the logger module to stdout -- okay, so I added that. When you change your mind after I spend the time to address the original comment -- it makes for an extremely annoying peer review experience. At this point, if I make more changes I'm not convinced you wont have more concerns about implementation after I write the code and complete all the testing.

I guess I'll start with saying it was a #protip for next time rather than a request, as I've explained and quoted myself above. Also, you didn't just switch to use `logging` in https://github.com/freedomofpress/securedrop/pull/1781/commits/424a2c76299c05a12745aa5c506e2c95786ce0d2, you also added a whole bunch of status messaging (stdout) in addition to the error messaging that already existed (stderr). I should have said `sys.stderr` earlier, and not `sys.stdout`.

>I have to insist at this point that you either modify the script to meet your specifications and push to this PR (make sure you test in Tails first) or give me a more succinct list of exactly what you have in mind and we can come to a compromise.

Again... I feel like this isn't taken into context our Slack convo.